setup.cfg

```diff
@@ -37,6 +40,143 @@ packages =
     zmq
 resources =
     artwork
+
+[py2exe]
+dll_excludes =
+    crypt32.dll,
+    iphlpapi.dll,
+    jvm.dll,
+    kernelbase.dll,
+    mpr.dll,
+    msasn1.dll,
+    msvcm90.dll,
+    msvcp90.dll,
+    msvcr90.dll,
+    nsi.dll,
+    uxtheme.dll,
+    winnsi.dll
+excludes =
+    Cython,
+    ilastik,
+    IPython,
+    matplotlib.backends.backend_cairo,
+    matplotlib.backends.backend_qt*,
+    matplotlib.backends.backend_tk*,
+    pylab,
+    Tkinter,
+    vigra
+includes =
+    h5py,
+    h5py.*,
+    lxml,
+    lxml.*,
+    scipy.io.matlab.streams,
+    scipy.linalg.*,
+    scipy.special,
+    scipy.special.*,
+    scipy.sparse.csgraph._validation,
+    skimage.draw,
+    skimage._shared.geometry,
+    skimage.filters.rank.*,
+    sklearn.*,
+    sklearn.neighbors,
+    sklearn.neighbors.*,
+    sklearn.utils.*,
+    sklearn.utils.sparsetools.*
+packages =
+    cellprofiler,
+    cellprofiler.cpmath,
+    cellprofiler.gui,
+    cellprofiler.icons,
+    cellprofiler.matlab,
+    cellprofiler.modules,
+    cellprofiler.utilities,
+    centrosome,
+    contrib,
+    h5py,
+    imagej,
+    libtiff,
+    lxml
+skip_archive = 1
+
+[py2exe_ilastik]
```

Comment:
I think moving `[py2exe_ilastik]` out of master is ok for now - I can work on it on a branch.

There's Ilastik the application and there's Ilastik the library that's needed for classify_pixels to run. In an ideal world, you'd have "ilastik==0.5" in the "install_requires" list and then we'd modify the `[py2exe]` section to look like `[py2exe_ilastik]`. Packaging the Ilastik dependency, Vigra so it can be pip installed would be very difficult - building Vigra requires CMake and installation of system packages like libhdf5. Until we get to that, I am hoping we can have a non-standard (but reproducible) build process for Ilastik and its dependencies, a command that packages a binary executable that can run classify_pixels and a setup script that allows CellProfiler to be pip installed on a bare Python without Ilastik support.

But yes - fine to have a CellProfiler distribution without the Ilastik application as long as we can provide a version of Ilastik that can produce classifiers compatible with classify_pixels.